Student/StudentUITests/Tests/StudentTest.swift

```diff
@@ -89,11 +89,22 @@ class StudentTest: XCTestCase {
         }
     }
 
-    func allowAccessToMicrophone() {
-        let alert = app!.alerts["“Student” Would Like to Access the Microphone"]
-        if alert.exists {
-            alert.buttons["OK"].tap()
+    func allowAccessToMicrophone(waitFor id: String, afterRunning activate: () -> Void) {
+        let alertHandler = addUIInterruptionMonitor(withDescription: "Permission Alert") { (alert) -> Bool in
```

Comment:
What in the world is `addUIInterruptionMonitor`?

Reply:
Do we need to use it for camera?

Reply:
This is the handler that can actually see system dialogs like this since earl gery doesn't seem to see them, nor does the XCUIApplication.

Reply:
I haven't seen the same problem with the camera permission dialog, but I don't know if it could benefit from using the same pattern.